### PR TITLE
Fetaure/support manual signing of extensions

### PR DESCRIPTION
**Pull request by Neil3079**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ For more detailed instructions [see our wiki](https://github.com/theappbusiness/
 
 For more detailed information on how to setup your project and environment please see our [wiki](https://github.com/theappbusiness/MasterFastfile/wiki)
 
+## Provfiles
```
Comment (lucianomarisi) [inline README.md]: profiles typo

File: Fastfile [hunk 2]
```diff
@@ -54,13 +53,16 @@ default_platform :ios
...
     if ENV['TAB_EXPORT_METHOD'] != nil
       export_method = ENV['TAB_EXPORT_METHOD']
     else
       UI.message("Fallbacking to enterprise export_method sinceTAB_EXPORT_METHOD is not defined")
     end
-    if provisioning_profile_name != nil
-      xcconfig_filename = Dir.pwd + "/TAB.release.xcconfig"
+    if File.file?("Provfile")
+      _parse_provision_file()
+      gym(use_legacy_build_api: true, xcconfig: xcconfig_filename)
```
Comment (lucianomarisi) [inline Fastfile]: I thought we should no longer use the legacy build api?

**Review by lucianomarisi — APPROVED** [review]

**Comment by lucianomarisi** [comment]
do we need to update the CHANGELOG?

**Comment by Neil3079** [comment]
This needs some work

**Pull request closed without merging**